### ISSUE TITLE
Jump to source opening a blank buffer

### ISSUE THREAD

**Issue by b-ryan** [liquidz/vim-iced#405]
Hello!

vim-iced version: 2.7.6
neovim version:

```NVIM v0.6.1
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by team+vim@tracker.debian.org

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

Jump to source was working, but then I upgraded both vim-iced and neovim. I am not sure which one may have caused this behavior to stop working.

I created a new project to test this, using `lein new app lein-app`. I then started `iced repl`, connected, and tried to jump to the `println` function source. It opens a blank buffer.

The buffer that's open is for `~/src/_scratch/lein-app/zipfile:/home/buck/.m2/repository/org/clojure/clojure/1.10.3/clojure-1.10.3.jar::clojure/core.clj`.

If I use neovim to open this JAR manually, it opens `zipfile:///home/buck/.m2/repository/org/clojure/clojure/1.10.3/clojure-1.10.3.jar::clojure/core.clj`. I don't know if this is relevant.

**Comment by liquidz** [comment]
@b-ryan Thanks for your reporting!
I could reproduce the problem with neovim v0.7.0 and vim-iced v2.7.6.
And neovim v0.7.0 and vim-iced v3.8.7(latest) has no problem.

The problem should have been solved by vim-iced v3.7.4.
* https://github.com/liquidz/vim-iced/blob/main/CHANGELOG.adoc#374-2022-01-09
* https://github.com/liquidz/vim-iced/commit/dbbf2892fd1881aab4e653f74849dec1a67f7193

**Comment by b-ryan** [comment]
Ah thank you! I did not realize I had pinned my vim-iced to version 2.7.6 in my vim init script. I properly upgraded this time and now it's working.
